libbeat/idxmgmt/std.go

```diff
@@ -314,9 +299,7 @@ func getEventCustomIndex(evt *beat.Event, beatInfo beat.Info) string {
 	}
 
 	if idx, err := events.GetMetaStringValue(*evt, events.FieldMetaIndex); err == nil {
-		ts := evt.Timestamp.UTC()
-		return fmt.Sprintf("%s-%d.%02d.%02d",
-			strings.ToLower(idx), ts.Year(), ts.Month(), ts.Day())
+		return strings.ToLower(idx)
```

Comment:
I noticed the comment on `FieldMetaIndex` is now out of date. This looks like the only usage of it.

```go
	// FieldMetaIndex defines the base index name to use for the event. The value is suffixed
	// with a Y-m-d value based on the event's timestamp. If set, it takes precedence over the
	// value defined using FieldMetaRawIndex.
	FieldMetaIndex = "index"
```

Can you remove the reference to the `Y-m-d` suffix here? https://github.com/elastic/beats/blob/master/libbeat/beat/events/util.go#L33